Task7_1/HttpModule.Task7_1.SampleApp/Program.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections.Generic;
+using HttpModule.Task7.SiteDownloader.Model.Constraint;
+using HttpModule.Task7.SiteDownloader.Services;
+using HttpModule.Task7.SiteDownloader.Services.Abstractions;
+using HttpModule.Task7.SiteDownloader.Services.EventArgs;
+using McMaster.Extensions.CommandLineUtils;
+using Console = System.Console;
+
+namespace HttpModule.Task7_1.SampleApp
+{
+    public class Program
+    {
+        [Option(CommandOptionType.SingleValue, ShortName = "u")]
+        public string SiteUrl { get; set; }
+
+        [Option(CommandOptionType.SingleValue, ShortName = "o")]
+        public string OutputDirectory { get; set; }
+
+        [Option(CommandOptionType.SingleValue, ShortName = "d")]
+        public int MaxDepth { get; set; } = 1;
+
+        [Option(CommandOptionType.SingleValue, ShortName = "c")]
+        public CrossDomainConstraintType CrossDomainConstraintType { get; set; } =
+            CrossDomainConstraintType.OnlyCurrentDomain;
+
+        [Option(CommandOptionType.SingleValue, ShortName = "l")]
+        public bool Logging { get; set; } = true;
+
+        [Option(CommandOptionType.MultipleValue, ShortName = "e")]
+        public IEnumerable<string> AllowedExtensions { get; set; }
+
+        public static int Main(string[] args)
+            => CommandLineApplication.Execute<Program>(args);
```

Comment:
Please handle case when user just runed this app without any args

Reply:
Done